src/components/UI/PageHeading.tsx

```diff
@@ -0,0 +1,10 @@
+import { Heading } from '@chakra-ui/react';
+import { FC } from 'react';
+
+export const PageHeading: FC = ({ children }) => {
+  return (
+    <Heading as='h1' color='brand.heading' fontSize='h1' fontWeight={200} lineHeight='48px' mb={4}>
```

Comment:
For some reason, this heading is not using the font "Libre Franklin" that you have declared in the theme.
![image](https://user-images.githubusercontent.com/468158/147511464-607cbf27-0ab6-4771-9aae-3ef94921f53e.png)

Reply:
It seems to be working fine on my browser, it's using Libre Franklin

<img width="327" alt="Screen Shot 2021-12-27 at 21 28 55" src="https://user-images.githubusercontent.com/948922/147515576-7ad95d26-edc7-4b26-8a17-73b35f8220a6.png">

Reply:
Oh ok nice then. I was comparing it with the design where we might be using a smaller font weight:
![image](https://user-images.githubusercontent.com/468158/147564411-9af607eb-9a98-435e-ae35-31a251ff17c5.png)

Reply:
Yes, it should be font-weight 200

Reply:
The `F` is glitched in the design on figma, all the text should be font-weight 200 as you said